lasagne/layers/cuda_convnet.py

```diff
@@ -13,7 +13,16 @@
 from .pool import pool_output_length
 from ..utils import as_tuple
 
-from theano.sandbox.cuda.basic_ops import gpu_contiguous
+# check if Theano's old GPU backend is available and in use
+try:
+    from theano.sandbox.cuda.basic_ops import gpu_contiguous
```

Comment:
This doesn't check whether it's in use. `import lasagne.layers.cuda_convnet` should also fail on Theano 0.8 or 0.9 with `device=cuda`, not only on Theano 0.10.